src/cmd/team.go

```diff
@@ -88,6 +88,55 @@ var createContactCmd = &cobra.Command{
 	},
 }
 
+var createTeamTagCmd = &cobra.Command{
+	Use:   "tag ID|ALIAS TAG_KEY TAG_VALUE",
+	Short: "Create a team tag",
+	Long: `Create a team tag
+	
+opslevel create team tag my-team foo bar
+opslevel create team tag --assign my-team foo bar
+`,
+	Args:       cobra.ExactArgs(3),
+	ArgAliases: []string{"ID", "ALIAS", "TAG_KEY", "TAG_VALUE"},
+	Run: func(cmd *cobra.Command, args []string) {
+		var result interface{}
+		var err error
+		teamKey := args[0]
+		tagKey := args[1]
+		tagValue := args[2]
+		tagAssign, err := cmd.Flags().GetBool("assign")
+		cobra.CheckErr(err)
+		if tagAssign {
+			input := opslevel.TagAssignInput{
+				Tags: []opslevel.TagInput{
+					{Key: tagKey, Value: tagValue},
+				},
+			}
+			if common.IsID(teamKey) {
+				input.Id = teamKey
+			} else {
+				input.Alias = teamKey
```

Comment:
If we run through this usecase we won't properly assign the tag because we need to set the `TaggableResource` type

Reply:
It does work if we use the alias since on line [120](https://github.com/OpsLevel/cli/pull/72/files#diff-8becba81854ddbaceaee2adba092fff02e24b2d565d93a056623285ee25b7086R120) we're setting the type to be `TaggableResourceTeam`

<img width="723" alt="Screen Shot 2022-11-11 at 1 11 17 PM" src="https://user-images.githubusercontent.com/33187974/201403030-0cbef6b5-a49a-49df-ae5d-3f848c41c82d.png">

Reply:
😵‍💫  sorry - you are correct.